d2plugin/plugin.go

```diff
@@ -132,12 +132,12 @@ func ListPluginInfos(ctx context.Context, ps []Plugin) ([]*PluginInfo, error) {
 }
 
 // FindPlugin finds the plugin with the given name.
-// 1. It first searches the bundled plugins in the global plugins slice.
-// 2. If not found, it then searches each directory in $PATH for a binary with the name
-//    d2plugin-<name>.
-//    **NOTE** When D2 upgrades to go 1.19, remember to ignore exec.ErrDot
-// 3. If such a binary is found, it builds an execPlugin in exec.go
-//    to get a plugin implementation around the binary and returns it.
+//  1. It first searches the bundled plugins in the global plugins slice.
+//  2. If not found, it then searches each directory in $PATH for a binary with the name
+//     d2plugin-<name>.
+//     **NOTE** When D2 upgrades to go 1.19, remember to ignore exec.ErrDot
```

Comment:
@nhooyr

Reply:
Confirmed we're good here, we shouldn't be ignoring exec.ErrDot, I was wrong.

Reply:
I'm working on a PR to remove this comment and fix a few other things about d2plugin.